docs/LP rewards/proposed_architecture/0x_example.md

```diff
@@ -0,0 +1,17 @@
+# Reward distribution for Gauges
+
+## example 1: lock on day 0, unlock on day 15
+| Gauge       | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10| 11| 12| 13| 14| 15| 16| 17| 18| 19| 20| 21| 22| 23| 24| 25| 26| 27| 28| 29|
+|----------   |---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|
+| 1 ( 1 day ) | - | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | - |
+| 2 ( 7 days) | - | x | x | x | x | x | x | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | O | x | x | x | x | x | x | - |
+| 3 (14 days) | - | x | x | x | x | x | x | x | x | x | x | x | x | x | O | O | x | x | x | x | x | x | x | x | x | x | x | x | x | - |
+
```

Comment:
Assuming its a 14 day lockup.
Then it should be
Gauge 1 has rewards on epoch 0 end / epoch 1 start until epoch 27 end / epoch 28 start. (It gets no rewards at epoch 28 end / epoch 29 start)

Gauge 2 has rewards on epoch 0 end / epoch 1 start, until epoch 21 end / epoch 22 start. (It gets no rewards at epoch 22 end)

Gauge 3 has rewards on epoch 0 end / epoch 1 start, and none on epoch 15 end. 

(I specify start/end, since I'm not sure if the day delimiters are marking rewards at beginning of the day, or at the end of the day)